app/Global.scala

```diff
@@ -0,0 +1,34 @@
+import de.zalando.play.controllers.PlayBodyParsing
+import play.api._
+import play.api.mvc.Results._
+import play.api.mvc._
+
+import scala.concurrent.Future
+
+/**
+  * The purpose of this object is to override default play's error reporting with application/json content type.
+  */
+object Global extends GlobalSettings {
```

Comment:
I'm not sure that Global approach is deprecated. But ```DefaultHttpErrorHandler```is cleaner approach, definitely. Thanks.